scripts/conf/tracing.properties

```diff
@@ -2,6 +2,7 @@ proxy_host=
 proxy_port=
 proxy_user_name=
 proxy_password=
+proxy_secure=false
 hostname_alias=
 # sender uses https by default to send traces; to switch to http, just change protocol prefix to http://
```

Comment:
Given this comment here, should the default for your new `proxy_secure` be `true`?

Reply:
Same in the `agent.properties`.

Reply:
No I think better to have it as false

Reply:
Can you clarify?

Reply:
Even proxy is used without security, you can still use https in the endpoints.  
So this would happen;
```
agent -----without-secure-----proxy-----secure-------SC
```
I see secure=false could be better because probably most users have proxy in their local, over a local ip, without a SSL cert.